src/containers/my-resources/categories-container/CategoriesContainer.tsx

```diff
@@ -96,9 +96,14 @@ const CategoriesContainer = () => {
     []
   )
 
+  const deleteCategory = useCallback(
+    (id?: string) => ResourceService.deleteResourceCategory(id ?? ''),
```

Comment:
What should happen if there is no category ID ?

Reply:
1) If there is no category ID user received the error message

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/db3e032c-85e3-4b40-a3cc-ffcef2e716d7

2) MyResourcesTable container has methods for all My Resources tabs (lessons, questions, attachments, categories) to display notifications in case of delete
<img width="451" alt="Знімок екрана 2023-10-23 о 11 35 04" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/9d80038f-323e-4585-89c7-2f5ef68d776b">